src/norm/sqlite/sqlgen.nim

```diff
@@ -23,15 +23,15 @@ proc getTable*(objRepr: ObjRepr): string =
   result = objRepr.signature.name.toLowerAscii()
 
   for prag in objRepr.signature.pragmas:
-    if prag.name == "table" and prag.kind == pkKval:
+    if prag.name == "dbTable" and prag.kind == pkKval:
```

Comment:
Please check for both dbTable and table for the transition period. We'll remove it after a couple of minor versions. A TODO mark is fine.